### PR TITLE
Added ES6 support using babel

### PR DESCRIPTION
**Pull request by lobut**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,7 @@ reports
 node_modules
 .sass-cache
 .DS_Store
+
+### editors ###
+jsconfig.json
+.vscode
```
Comment (alexplatonov) [inline .gitignore]: forgot new line at the end of the file

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
   },
   "devDependencies": {
     "angular-mocks": "^1.4.8",
+    "babel-preset-es2015": "^6.1.18",
```
Comment (alexplatonov) [inline package.json]: Use ~ instead

File: src/core/scripts/app.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ angular.module('angularity', [
         'ngTouch',
         'ngRoute'
     ])
-    .config(function($httpProvider, $routeProvider, $locationProvider) {
+    .config(($httpProvider, $routeProvider, $locationProvider) => {
```
Comment (alexplatonov) [inline src/core/scripts/app.js:7]: ngAnnotatemight have problem with es6
Check this: https://github.com/olov/ng-annotate#es6-and-typescript-support

Comment (lobut) [inline src/core/scripts/app.js:7]: Well, then I can also convert all the top level controllers that use injection to use @ng-inject as well!

**Comment by lobut** [comment]
@alexplatonov can you take a look at this?

File: package.json.orig [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "angular": "~1.4.8",
+    "angular-cookies": "~1.4.8",
+    "angular-route": "~1.4.8",
+    "angular-sanitize": "~1.4.8",
+    "angular-touch": "~1.4.8"
+  },
+  "devDependencies": {
+<<<<<<< HEAD
```
Comment (alexplatonov) [inline package.json.orig]: merge conflicts?

Comment (alexplatonov) [inline package.json.orig]: What's that file anyway?

**Pull request merged**